rules/right-format-to-write-videos-time-length/rule.md

```diff
@@ -16,22 +16,22 @@ guid: 1210d482-f57a-4a0e-9fe4-c229ea93f4a4
 ---
 
 The following table shows the right format for writing video time lengths (hours, minutes, seconds) on the web.
-            
+
 <!--endintro-->
 
 | Duration | Correct format | ❌ Bad examples | ✅ Good example |
 |---|---|---|---|
 | Less than 1 minute | {{x}} sec | 30s / 30secs / 30 secs | 30 sec |
 | 1 to 59 minutes | {{x}} min | 25m / 25mins / 25 mins | 25 min |
-| 1 hour | {{x}} h | 60 mins / 1 hour / 1h 00m | 1 hr |
-| More then 1 hour | {{x}} h {{x}} m| 1h 05m / 1h5min / 1h 5m 10s | 1 hr 5 min |
+| 1 hour | {{x}} hr | 60 mins / 1 hour / 1h 00m | 1 hr |
+| More then 1 hour | {{x}} h {{x}} m| 1h 05m / 1h5min / 1h 5m 10s / 2 hrs | 1 hr 5 min / 2 hr |

```

Comment:
Change from
    | More then 1 hour | {{x}} h {{x}} m| 1h 05m / 1h5min / 1h 5m 10s / 2 hrs | 1 hr 5 min / 2 hr |
To 
    | More **_than_** 1 hour | {{x}} **_hr_** {{x}} **_min_** | 1h 05m / 1h5min / 1h 5m 10s / 2 hrs | 1 hr 5 min / 2 hr |

Reply:
Thanks, fixed!